modules/virt-creating-configmap.adoc

```diff
@@ -1,16 +1,9 @@
 // Module included in the following assemblies:
 // * virt/virtual_machines/importing_vms/virt-importing-rhv-vm.adoc
 [id='virt-creating-configmap_{context}']
```

Comment:
I know it's unrelated to the PR scope but can you please insert a newline between the metadata and the id. It's important for managing module metadata at scale.
And the quote marks in the id should be ":
[id="virt-creating-configmap_{context}"]

For reference:
https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#module-file-metadata

Reply:
OK, but there is no good reason for either of these changes. Not having a line between metadata and ID does not create a problem. Same with quotes.

Also, this module will be removed in CNV 2.5.